tracing/src/lib.rs

```diff
@@ -669,6 +670,60 @@
 //!
 //! [log-tracer]: https://docs.rs/tracing-log/latest/tracing_log/#convert-log-records-to-tracing-events
 //!
+//! ### `no_std` Support
```

Comment:
should we just link to core's docs here rather than repeating?

Reply:
i think it's likely that more people will be reading the `tracing` docs than the `tracing-core` ones, so i thought it was better to reproduce it here...@davidbarsky wdyt?

Reply:
I'd rather reproduce here for now.